src/tcp_wrap.cc

```diff
@@ -77,13 +77,9 @@ void TCPWrap::Initialize(Local<Object> target,
   // Init properties
   t->InstanceTemplate()->Set(String::NewFromUtf8(env->isolate(), "reading"),
```

Comment:
Any reason not to add `"reading"` to `env.h` as well? Are we trying to keep that list small?